examples/kotlin/hello_world/tools/android-studio-run-configurations/run_configuration_example_debug_arm64.xml

```diff
@@ -0,0 +1,38 @@
+<configuration name="Example App (Debug) [arm64]"
```

Comment:
this configuration is basically a copy-paste of the existing x86 run configuration